mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/mapping/building/spi/PojoMappingCollectorValueNode.java

```diff
@@ -19,4 +19,5 @@ void valueBridge(BridgeBuilder<? extends ValueBridge<?, ?>> builder,
 
 	void indexedEmbedded(String relativePrefix, ObjectFieldStorage storage, Integer maxDepth, Set<String> includePaths);
 
+	void defaultScale(int scale);
```

Comment:
This doesn't have anything to do with the mapping, since it doesn't create anything in the index schema. This is purely Pojo metadata and should be moved to `PojoAdditionalMetadataCollectorValueNode`.